demos/mlsys_2020/demo_handler.py

```diff
@@ -0,0 +1,137 @@
+from demo_source import *
```

Comment:
It's generally frowned upon in the python world to use `from module import *`.

https://docs.python.org/2/tutorial/modules.html#more-on-modules

> Note that in general the practice of importing * from a module or package is frowned upon, since it often causes poorly readable code. However, it is okay to use it to save typing in interactive sessions.